### PR TITLE
Adds CMake option to enable/disable tests, default enabled when building standalone

### PR DESCRIPTION
**Pull request by robiwano**

### CODE REVIEW

**Comment by erikd** [comment]
This does not pass CI. If you fix it, I will be merged.

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.1)
 project(libsamplerate VERSION 0.1.9 LANGUAGES C)
 
+option(LIBSAMPLERATE_TESTS ON "Enable to generate test targets")
```
Comment (qno) [inline CMakeLists.txt]: setting this option is wrong - it must be changed to 

`option(LIBSAMPLERATE_TESTS "Enable to generate test targets" ON)`

see https://cmake.org/cmake/help/latest/command/option.html

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+option(LIBSAMPLERATE_INSTALL ON "Enable to add install directives")
```
Comment (qno) [inline CMakeLists.txt]: dito

File: CMakeLists.txt (hunk 1 above)
Comment (robiwano) [reply CMakeLists.txt]: Indeed, thanks!

**Comment by erikd** [comment]
@robiwano Would you be able to squash these two commits into a single one?

**Comment by erikd** [comment]
Applied. Thanks!

**Pull request closed without merging**